README.md

```diff
@@ -226,7 +226,7 @@ In summary, you:
 [Check out the COLAB demo here](https://colab.research.google.com/drive/1F_RNcHzTfFuQf-LeKvSlud6x7jXYkG31#scrollTo=HOk9c4_35FKg)
 
 ## What types of research works?
-Anything! Remember, that this is just organized PyTorch code.
+Anything! Remember, that this is just an organized PyTorch code.
```

Comment:
```suggestion
Anything! Remember, that this is just organized PyTorch code.
```

this is wrong...  (Use “a” before words that start with a consonant sound and “an” before words that start with a vowel sound). Py is not a vowel

Reply:
but I had put "an" before organized(starting with a vowel)

Reply:
```suggestion
Anything! Remember, that this is just organized PyTorch code.
```

IMO there is no article needed here...